devbox.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packages": [
+    "python@3.7",
+    "python37Packages.pip@latest",
+    "python37Packages.setuptools@latest",
+    "python37Packages.wheel@latest"
+  ],
+  "shell": {
+    "init_hook": [
+      "[ -z \"$FISH_VERSION\" ] && . \"$VENV_DIR/bin/activate\" || . \"$VENV_DIR/bin/activate.fish\"",
+      "pip install -e .[dev]"
+    ],
+    "scripts": {
+      "test": [
+        "python -m pytest"
+      ]
+    }
+  }
+}
```

Comment:
Devbox doesn't currently write trailing newlines to their files 😢 

I have an issue open to fix this upstream: https://github.com/jetpack-io/devbox/issues/1248